README.md

```diff
@@ -67,11 +63,15 @@ We recommend you use the "adal" tag so we can see it! Here is the latest Q&A on
 
 ## Security Reporting
 
-If you find a security issue with our libraries or services please report it to [secure@microsoft.com](mailto:secure@microsoft.com) with as much detail as possible. Your submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) program. Please do not post security issues to GitHub Issues or any other public site. We will contact you shortly upon receiving the information. We encourage you to get notifications of when security incidents occur by visiting [this page](https://technet.microsoft.com/en-us/security/dd252948) and subscribing to Security Advisory Alerts.
+See [Security.md](./Security.md)
```

Comment:
Seems like a dead link. Security.md -> SECURITY.md?

Reply:
Thank you for catching this!